### PR TITLE
Add and delete queries

### PR DESCRIPTION
**Pull request by cndonovan**
Implement adding and deleting queries

Screen Shot 2022-10-17 at 4 12 38 PM
Screen Shot 2022-10-17 at 4 12 58 PM

Signed-off-by: Christine Donovan <christine.donovan@chronosphere.io>

### CODE REVIEW

**Comment by Nexucis** [comment]
Nice, thanks for the changes !

Do you think it make sense to have a separation between each query ? Like an horizontal line, or having each query in a sort of card.

File: ui/dashboards/src/components/PanelDrawer/PanelEditorForm.tsx [hunk 1]
```diff
@@ -71,7 +71,8 @@ export function PanelEditorForm(props: PanelEditorFormProps) {
   };
 
   return (
-    <form id={panelEditorFormId} onSubmit={handleSubmit}>
+    // Padding accounts for the combination of overflow: 'auto' and the MUI input labels - the labels are cut off otherwise.
+    <form id={panelEditorFormId} onSubmit={handleSubmit} style={{ overflowY: 'auto', padding: '8px 0' }}>
```
Comment (eunicorn) [inline ui/dashboards/src/components/PanelDrawer/PanelEditorForm.tsx:75]: maybe use the `styled` function from mui so we can use `theme.spacing`

**Review by eunicorn — APPROVED** [review]

**Comment by cndonovan** [comment]
> Nice, thanks for the changes !
> 
> Do you think it make sense to have a separation between each query ? Like an horizontal line, or having each query in a sort of card.

Oh yeah good point -- that's in designs so I'll add that in the nearish future (the horizontal line)

File: ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartOptionsEditor.tsx [hunk 2]
```diff
@@ -12,16 +12,28 @@
...
+import { OptionsEditorProps, TimeSeriesQueryEditor, usePlugin } from '@perses-dev/plugin-system';
+
 import { TimeSeriesChartOptions } from './time-series-chart-model';
 
+const DEFAULT_QUERY_PLUGIN_TYPE = 'TimeSeriesQuery';
+const DEFAULT_QUERY_PLUGIN_KIND = 'PrometheusTimeSeriesQuery';
+
 export type TimeSeriesChartOptionsEditorProps = OptionsEditorProps<TimeSeriesChartOptions>;
 
 export function TimeSeriesChartOptionsEditor(props: TimeSeriesChartOptionsEditorProps) {
```
Comment (sjcobb) [inline ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartOptionsEditor.tsx:28]: Oh man, this file is about to get huge after my [legend options PR](https://github.com/perses/perses/pull/660/files#diff-bfad4a5ad2e72f83d687801cdb2b390252da88da4dcd59ddf3b027c0c359c514) gets merged in 😂 (I didn't add the 'Query' and 'Visualization' tabs yet so it'll get better after that)

**Review by sjcobb — APPROVED** [review]
🚀

**Comment by cndonovan** [comment]
This is ready to merge if anyone wants to, and I can clean up the scrolling bit in a different PR

**Pull request merged**